project/Dependencies.scala

```diff
@@ -27,13 +27,12 @@ object Dependencies {
   val commonsHttpClient = "commons-httpclient" % "commons-httpclient" % "3.1"
   val commonsIo = "commons-io" % "commons-io" % "2.4"
   val cssParser = "net.sourceforge.cssparser" % "cssparser" % "0.9.18"
-  val contentApiClient = "com.gu" %% "content-api-client" % "8.2"
+  val contentApiClient = "com.gu" %% "content-api-client" % "8.10"
   val dfpAxis = "com.google.api-ads" % "dfp-axis" % "2.8.0"
   val dispatch = "net.databinder.dispatch" %% "dispatch-core" % "0.11.3"
   val dispatchTest = "net.databinder.dispatch" %% "dispatch-core" % "0.11.3" % Test
   val exactTargetClient = "com.gu" %% "exact-target-client" % "2.24"
   val faciaFapiScalaClient = "com.gu" %% "fapi-client" % faciaVersion
-  val faciaScalaClient = "com.gu" %% "facia-json" % faciaVersion
```

Comment:
`facia-json` is pulled as a dependency of `fapi-client`, so there is not need to add it explicitly. Let me know if I wrongly assume that